packages/core-flows/src/inventory/steps/attach-inventory-items.ts

```diff
@@ -0,0 +1,43 @@
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+import { ContainerRegistrationKeys } from "@medusajs/utils"
+import { InventoryItemDTO } from "@medusajs/types"
+
+export const attachInventoryItemToVariantsStepId =
+  "attach-inventory-items-to-variants-step"
+export const attachInventoryItemToVariants = createStep(
+  attachInventoryItemToVariantsStepId,
+  async (
+    input: {
+      inventoryItem: InventoryItemDTO
```

Comment:
todo: here we are asking for the entire DTO but we are only using th id, can we update the type to not be required to pass everything and more constraint to what we are expecting instead to reduce coupling.